### ISSUE TITLE
Hard to follow muted people's conversation

### ISSUE THREAD

**Issue by xDashh** [BryceMatthes/betterdgg#55]
When a user gets muted in chat, his message and name get grayed out. 
If you click on their name after that, it is (too) hard to follow their conversaiton; colors should be adjusted.

muted: https://i.imgur.com/FAHg7Dh.png
muted and "highlighted": https://i.imgur.com/2YuXFfN.png

**Comment by PinkishCow** [comment]
Tried to do this and caught every disease that had inhabited the earth in the past eon. You should probably reopen it PEPE

**Comment by downthecrop** [comment]
Seems that #67 fixes this issue.

Muted message

![image](https://cloud.githubusercontent.com/assets/4764745/18531027/6df7c00e-7a91-11e6-90ac-0f3c370b3b1d.png)

Highlighted muted user

![image](https://cloud.githubusercontent.com/assets/4764745/18531049/89254cc0-7a91-11e6-950e-8ea3556e490b.png)
